### PR TITLE
Meta EIP: decentralize EIP workflow

### PR DESCRIPTION
**Pull request by ethernian**
Dear Editors,
this EIP currently outlines a new process, leaving out implementation details. I hope it is enough for now to start a meaning discussion. Further discussion is planned on ethereum-magicians gathering in Prague.

Few additions:
My github name (dip239) differs from my name in ethereum space (Ethernian). An Email address <dima@ethernian.com> should be matched.

Regards, Dimi

### CODE REVIEW

File: EIPS/eip-XXX-decentralize-EIP-workflow.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
+eip: <to be assigned>
```
Comment (Arachnid) [inline EIPS/eip-XXX-decentralize-EIP-workflow.md]: Please format this as per the instructions in EIP 0.

Comment (axic) [reply EIPS/eip-XXX-decentralize-EIP-workflow.md]: > Please format this as per the instructions in EIP 0.

As per EIP 1.

**Comment by axic** [comment]
@ethernian I think this would rather be an update to EIP-1 to change the rules defined in there?

Would it make sense closing this PR for now and keeping the discussion on ethmagi and coming back here when the proposal is more crystallised?

**Comment by ethernian** [comment]
It is highly unlikely to see progress here before the dapplets project become stable, so it could be cleaned up.

**Pull request closed without merging**